cli/api.py

```diff
@@ -126,7 +124,8 @@ def generate_api(input_dir, output, summary_output, aggregation_level, state, fi
 
     active_states = [state.abbr for state in us.STATES]
     active_states = active_states + ["PR", "MP"]
-    regions = combined_datasets.get_subset_regions(
+
+    selected_dataset = combined_datasets.load_us_timeseries_dataset().get_subset(
```

Comment:
I wonder when the right time to remove this code is.  Now that the CRS is using the new api, maybe soon?  Also may be good to refresh api v1 once before we remove the code

Reply:
Sounds good to me.